docs/MixPanel-batchsource.md

```diff
@@ -0,0 +1,22 @@
+# MixPanel batch source
+
+Description
+-----------
+This plugin used to query MixPanel events.
+
+Properties
+----------
+### General
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**API Secret:** API secret to be used to authenticate in MixPanel API.
```

Comment:
API secret used to authenticate in MixPanel API.

Reply:
done